packages/client/components/DashboardAvatars/DashboardAvatars.tsx

```diff
@@ -64,93 +42,104 @@ const StyledButton = styled(PlainButton)({
   WebkitTapHighlightColor: 'transparent',
   '&:hover': {
     cursor: 'pointer'
-  }
+  },
+  marginTop: sizeToHeightBump[ElementWidth.DASHBOARD_AVATAR] * -1 // coupling with AvatarList styles
 })
 
 interface Props {
   team: DashboardAvatars_team
 }
 
-type Avatar = DashboardAvatars_team['teamMembers'][0]
+type Avatar = DashboardAvatars_team['teamMembers'][0]['user']
```

Comment:
-1 this type is now the user rather than the avatar so we should update the name. We also need to update the names in the logic below to reflect this